tests/store/test_emissions.py

```diff
@@ -11,43 +13,67 @@
 from helpers import clear_test_stores
 
 
-def test_read_binary_data(mocker):
-    clear_test_stores()
-    fake_uuids = ["test-uuid-1", "test-uuid-2", "test-uuid-3"]
-    mocker.patch("uuid.uuid4", side_effect=fake_uuids)
+# fixtures
+@pytest.fixture
+def bucket():
+    return get_bucket()
 
-    test_datapath = get_emissions_datapath("co2-gpp-cardamom_EUROPE_2012.nc")
 
-    binary_data = test_datapath.read_bytes()
+@pytest.fixture
+def standardise(bucket):
+    """Use `standardise(**kwargs)` to standardise emissions data.
 
-    metadata = {
-        "species": "co2",
-        "source": "gpp-cardamom",
-        "domain": "europe",
-        "high_time_resolution": False,
-    }
+    We only deal with emissions data in this test module, and most
+    tests just use `bucket = get_bucket()`, so this works for
+    most tests in this module.
+    """
+    return partial(standardise_flux, bucket=bucket)
+
+
+class TestCardamom:
```

Comment:
I understand why this approach has been taken to create test classes to group together functionality but if this needs to be split out in this way, this may be better split into two separate files instead, linking with the appropriate fixtures. If possible it would be preferable to keep this as a set of tests in one file, linking to fixtures if different data needs to be set up.

A few other points here:
 - Often the data is used not because it's EDGAR / Cardamom data but as a way to simulate a particular set of steps. So splitting by the two emissions types here doesn't necessarily map to the purpose of the tests.
 - As long as we understand what we're testing, having both EDGAR and Cardamom data within the same object store is not a wholly bad thing (as it could highlight some problems we hadn't anticipated) - however I do appreciate this is a less clear and predictable approach for the tests.

Reply:
I think I did this because I figured the groups made sense. I'll just drop the classes if the groups don't make sense. (Some book I have suggested using classes for groups of related tests. They said they don't typically use them for grouping fixtures, but it seemed to make sense in this case.)